### PR TITLE
add stuff

### PR DESCRIPTION
**Pull request by swatkatz**

### CODE REVIEW

**Comment by Alan-Cha** [comment]
@swatkatz Thanks for submitting this PR! Is this PR a WIP?

**Comment by swatkatz** [comment]
Whoops sorry, I think I clicked some wrong button. My bad.

**Comment by swatkatz** [comment]
I was just playing around on my cloned repo...sorry for the noise!

File: packages/openapi-to-graphql/src/auth_builder.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ type Viewer<TSource, TContext, TArgs> = {
   description: string
 }
 
-const translationLog = debug('translation')
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/src/auth_builder.ts:40]: @swatkatz Okay, no problem! Just to let you know, you can produce the debug messages by setting your environment variables.

For example, if you append `DEBUG=translation` followed by whatever you want to do, e.g. `DEBUG=translation node myGraphQLServer.js`, you should be able to get all the `translation` messages.

**Pull request closed without merging**